### ISSUE TITLE
The setup-tortuga script can fail but still return 0

### ISSUE THREAD

**Issue by cameronbrunner** [UnivaCorporation/tortuga#428]
Example from a fresh install:
```
Completing installer configuration. Please wait... done.
2018-12-17 19:35:37,147 - tortuga.setup - ERROR - Puppet post-configuration failed (see log file /tmp/tortuga_setup.log)
Puppet post-configuration failed (see log file /tmp/tortuga_setup.log)
2018-12-17 19:35:37,147 - tortuga.setup - ERROR - Fatal error occurred during setup
Traceback (most recent call last):
  File "/opt/tortuga/lib64/python3.6/site-packages/tortuga/deployer/tortugaDeployer.py", line 693, in runSetup
    self.puppetApply()
  File "/opt/tortuga/lib64/python3.6/site-packages/tortuga/deployer/tortugaDeployer.py", line 379, in puppetApply
    raise Exception(errmsg)
Exception: Puppet post-configuration failed (see log file /tmp/tortuga_setup.log)

Installation failed...
[root@installer ~]# echo $?
0
```

**Comment by dtougas** [comment]
I am surprised you didn't see this line printed out after the installation failed message: https://github.com/UnivaCorporation/tortuga/blob/master/src/installer/src/tortuga/scripts/tortuga_setup.py#L117

**Comment by dtougas** [comment]
This should be resolved by #430
